specs/specs.js

```diff
@@ -89,5 +89,32 @@ describe("Fingerprint2", function () {
         });
       });
     });
+	
+	describe("return components", function () {
+      it("does it return components", function (done) {
```

Comment:
should be imperative, e.g. it("returns components as a second argument to callback")